awscli/customizations/wizard/app.py

```diff
@@ -131,6 +138,18 @@ def _get_choices(self, value_name):
             return self._get_normalized_choice_values(choices)
         return None
 
+    def get_details_for_choice(self, choice):
+        step_definition = self._prompt_definitions[self._current_prompt]
+        if 'details' in step_definition:
+            value = self._convert_display_value_to_actual_value(
+                self._get_choices(self._current_prompt), choice
+            )
+            temp_values = self._values.copy()
+            temp_values[self._current_prompt] = value
+            details = temp_values[step_definition['details']['value']]
+            return json.dumps(details, indent=2,
```

Comment:
Were we planning to add the `format_type` to the `details` schema in this PR or in a follow up PR? I think it could be useful now so we are not always json dumping for this method by default.